lib/models/shows.js

```diff
@@ -0,0 +1,100 @@
+const fetch = require('node-fetch');
+
+async function getTrendingShows() {
+  try {
+    const response = await fetch(
+      `https://api.themoviedb.org/3/trending/tv/week?api_key=${process.env.TMDB_API_KEY}`
+    );
+    const trendingShows = await response.json();
+    const trendingShowsListView = trendingShows.results.map((show) => {
+      const trendingShowListView = {
+        id: show.id,
+        key: show.id,
+        title: show.name,
+        poster: `https://image.tmdb.org/t/p/w154/${show.poster_path}`,
+        rating: Math.floor(show.vote_average),
+      };
+      return trendingShowListView;
+    });
+    return trendingShowsListView;
+  } catch (error) {
+    throw new Error(error.message);
```

Comment:
If you just rethrow the error, you can remove the try-catch here. (happens multiple times)